src/constants/screenConstants.js

```diff
@@ -0,0 +1 @@
+export const XLARGE_SCREEN_WIDTH_LIMIT = 1140;
```

Comment:
XLARGE limit is not 1140, but 1200.
Resource:
https://getbootstrap.com/docs/4.1/layout/grid/#grid-options